.env.example

```diff
@@ -1,10 +0,0 @@
-REACT_APP_FIREBASE_API_KEY = 
```

Comment:
Hi, I think we should not delete this file. It's specifies the format for the .env to be actually used.

Reply:
Extra space but nvm that.